ats/emails.md

```diff
@@ -0,0 +1,34 @@
+# Emails API Endpoints
+
+
+* [Create Email](#post-email)
+
+
+---
+
+
+### `POST` `/email`
```

Comment:
I think the endpoint should actually be `/candidates/:candidateId/feed/emails`.
So instead of creating a new file, move this into candidates.md file